edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/EdgeHubDesiredProperties.cs

```diff
@@ -1,19 +1,29 @@
-// Copyright (c) Microsoft. All rights reserved.
-namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
-{
-    using System;
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System;
     using System.Collections.Generic;
     using Microsoft.Azure.Devices.Edge.Util;
-    using Newtonsoft.Json;
-
-    public class EdgeHubDesiredProperties
-    {
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// DTO that is used to deserialize EdgeHub Desired properties of the twin
+    /// into <see cref="EdgeHubConfig" /> by <see cref="TwinConfigSource" />.
+    /// </summary>
+    public class EdgeHubDesiredProperties
+    {
         [JsonConstructor]
-        public EdgeHubDesiredProperties(string schemaVersion, IDictionary<string, RouteConfiguration> routes, StoreAndForwardConfiguration storeAndForwardConfiguration)
+        public EdgeHubDesiredProperties(
+            string schemaVersion,
+            IDictionary<string, RouteConfiguration> routes,
+            StoreAndForwardConfiguration storeAndForwardConfiguration,
+            AuthorizationConfiguration authorizations)

```

Comment:
This should be mqttBroker configuration which contains authorizations and bridges (bridges will added later)

Reply:
Good catch, fixing this now.